### DISCUSSION TITLE
Questions about the NumberInput

### DISCUSSION THREAD

**Discussion by xiaoxiji1234** [lowdefy/lowdefy#1032] [Questions and Help]
Hi,  Can the NumberInput display prefixes and suffixes like this?
![image](https://user-images.githubusercontent.com/72787010/147054463-11a5f208-56df-408f-a154-17ef8f081a54.png)
Can the NumberInput look like Format the value like the InputNumber of ant.design?
<img width="581" alt="image" src="https://user-images.githubusercontent.com/72787010/147055054-a4da7715-7aef-4239-b7d8-cd6ae7e49409.png">

**Comment by SamTolmay** [comment]
Hi @xiaoxiji1234 

This is being implemented in V4, since we need to update the Ant Design library. You can see some more details here: #1019
